app/tests/integration/legacy/test_filter_expression_validation.py

```diff
@@ -1,3 +1,7 @@
+import pytest
+
+
+@pytest.mark.skip(reason="temporarily failing")
```

Comment:
Do we know why these are failing? That's where I left off my head-desking last night

Reply:
There's some weirdness in Firefox that causes this error. I dug into it a few times but there is no clear answer. I don't agree with this skip marker but I guess it is fine for now.

Reply:
Yeah just temporarily until we can fix them 👍